addons/mail_bot/static/src/js/notification_alert.js

```diff
@@ -3,20 +3,56 @@ odoo.define('mail.NotificationAlert', function (require) {
 
 var Widget = require('web.Widget');
 var widgetRegistry = require('web.widget_registry');
+var core = require('web.core');
 
+var _t = core._t;
 // -----------------------------------------------------------------------------
 // Display Notification alert on user preferences form view
 // -----------------------------------------------------------------------------
 var NotificationAlert = Widget.extend({
-   template: 'mail.NotificationAlert',
-   /**
-    * @override
-    */
-   init: function () {
-      this._super.apply(this, arguments);
-      var hasRequest = this.call('mailbot_service', 'isRequestingForNativeNotifications');
-      this.isNotificationBlocked = window.Notification && window.Notification.permission !== "granted" && !hasRequest;
-   },
+    template: "mail.NotificationAlert",
+    /**
+     * @override
+     */
+    init() {
+        this._super(...arguments);
+        const hasRequest = this.call("mailbot_service", "isRequestingForNativeNotifications");
+        this.isNotificationBlocked = window.Notification && window.Notification.permission !== "granted" && !hasRequest;
+        if (this.isNotificationBlocked) {
+            if (navigator.userAgent.includes("Chrome")) {
+                this.settingUrl = "chrome://settings/content/siteDetails?site=https%3A%2F%2F" + window.location.host;
+            } else if (navigator.userAgent.includes("Firefox")) {
+                this.settingUrl = "about:preferences#privacy";
+            } else if (navigator.userAgent.includes("Safari")) {
+                this.settingPath = "(path:Safari > Preferences > Website > Notifications)";
+            }
+        }
```

Comment:
All with single quotes, except the ones in sets of `settingUrl` and `settingPath`.